dotcom-rendering/src/palette.ts

```diff
@@ -4676,7 +4676,126 @@ const codeBlockBackgroundDark: PaletteFunction = () =>
 const codeBlockTextShadowLight: PaletteFunction = () =>
 	sourcePalette.neutral[100];
 const codeBlockTextShadowDark: PaletteFunction = () => sourcePalette.neutral[0];
-sourcePalette.neutral[38];
```

Comment:
this colour is not being used and appears to have been left in by mistake so I'm removing it